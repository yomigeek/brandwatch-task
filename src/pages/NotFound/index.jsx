import React from "react";
import styled from "styled-components";
import Header from "../../components/Header";
import Container from "../../components/Container";

const TextBox = styled.div`
  padding: 30px;
  font-size: 20px;
  text-align: center;
  color: #f26351;
`;


const NotFound = () => {
  return (
    <>
      <Header />
      <Container>
        <TextBox>
          <img
            src="https://res.cloudinary.com/kugoo/image/upload/v1631376228/webimages/404-review.png"
            alt="404-icon"
            height="150px"
          />
          <br />
          Oops! Page Not Found
          <br />
          <a href="/">Go home</a>
        </TextBox>
      </Container>
    </>
  );
};

export default NotFound;
