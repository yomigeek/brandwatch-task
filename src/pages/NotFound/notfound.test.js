/* eslint-disable no-undef */
import { render } from '@testing-library/react';
import NotFound from './index';

test('renders snapshot', () => {
  render(<NotFound />);
  const asFragment = render(<NotFound />).asFragment;
  expect(asFragment()).toMatchSnapshot();
});
