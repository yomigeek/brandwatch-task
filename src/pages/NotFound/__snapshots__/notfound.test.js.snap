// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot 1`] = `
<DocumentFragment>
  <header
    class="sc-bdvvtL eEUwUe"
  >
    <a
      href="/"
    >
      <img
        alt="logo"
        height="80px"
        src="https://res.cloudinary.com/kugoo/image/upload/v1645483046/upload/logo.png"
        width="80px"
      />
    </a>
  </header>
  <div
    class="sc-dkPtRN kFbBId"
  >
    <div
      class="sc-gsDKAQ lhdjnX"
    >
      <div
        class="sc-hKwDye heptGD"
      >
        <img
          alt="404-icon"
          height="150px"
          src="https://res.cloudinary.com/kugoo/image/upload/v1631376228/webimages/404-review.png"
        />
        <br />
        Oops! Page Not Found
        <br />
        <a
          href="/"
        >
          Go home
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
